src/app/core/entity/entity-remove.service.ts

```diff
@@ -29,8 +33,36 @@ export class EntityRemoveService {
     private entityMapper: EntityMapperService,
     private snackBar: MatSnackBar,
     private router: Router,
+    private schemaService: EntitySchemaService,
+    private fileService: FileService,
   ) {}
 
+  private showSnackbarConfirmation(
+    entity: Entity,
+    action: string,
+    navigateBackToUrl?: string,
+  ) {
+    const snackBarTitle = $localize`:Entity action confirmation message:${action} ${
```

Comment:
I don't think this part needs localize.

Reply:
may need to switch the order of action and type in different languages (in fact for most it needs to be the other way round, probably)